conda/recipes/cuspatial/meta.yaml

```diff
@@ -29,7 +29,7 @@ requirements:
   host:
     - python
     - cython >=0.29,<0.30
-    - cmake>=3.23.1
+    - cmake>=3.23.1,!=3.25.0
```

Comment:
Out of scope to fix in this PR, but the cuspatial and libcuspatial libraries should be aligned in terms of where they specify their required versions (conda_build_config.yaml vs meta.yaml) @thomcom @isVoid

Reply:
Thanks for the reminder, do you mind filing an issue?

Reply:
@gpucibot merge